base/scripts/objectives.cs

```diff
@@ -1013,17 +1056,22 @@ function checkObjectives(%this)
           
           $FlagIsDropped[%playerTeam] = false;
           $FlagIsDropped[%enemyTeam] = false;
-          
+
           $freeze::FlagClient[%playerTeam] = 0;
           $freeze::FlagClient[%enemyTeam] = 0;
           
           $freeze::OOB[%playerTeam] = false;
           $freeze::OOB[%enemyTeam] = false;
+          
+          Observer::CheckFlagStatus(0);
+          Observer::CheckFlagStatus(1);
 
           Item::hide(%flag, false);
           
+          GameBase::startFadeOut(%flag);
```

Comment:
does this do anything? the other fade outs are started 2.5 seconds before the timers run out - at this point the flag is already gone